### PR TITLE
doc: add gRPC proxy start doc

### PR DESCRIPTION
**Pull request by xiang90**
/cc @gyuho @heyitsanthony @fanminshi

### CODE REVIEW

File: Documentation/op-guide/grpc_proxy.md [hunk 1]
```diff
@@ -47,3 +47,31 @@ TODO
...
+```
+
+The etcd gRPC proxy starts and listens on port 8080. It forwards the requests to one of the three endpoints provided above.
+
+Try to send `PUT`, `GET` requests through the etcd gRPC proxy:
+
+```bash
+$ ETCDCTL_API=3 ./etcdctl --endpoints=127.0.0.1:8080 put foo bar
+OK
+$ ./etcdctl --endpoints=127.0.0.1:8080 get foo
```
Comment (heyitsanthony) [inline Documentation/op-guide/grpc_proxy.md]: `ETCDCTL_API=3`?

File: Documentation/op-guide/grpc_proxy.md [hunk 2]
```diff
@@ -47,3 +47,31 @@ TODO
...
+|infra1|10.0.1.11|infra1.example.com|
+|infra2|10.0.1.12|infra2.example.com|
+
+Start the etcd gRPC proxy to use these static endpoints with the command:
+
+```bash
+$ etcd grpc-proxy start --endpoints=infra0.example.com,infra1.example.com,infra2.example.com --listen-addr=127.0.0.1:8080
+```
+
+The etcd gRPC proxy starts and listens on port 8080. It forwards the requests to one of the three endpoints provided above.
```
Comment (heyitsanthony) [inline Documentation/op-guide/grpc_proxy.md]: `It forwards client requests`

File: Documentation/op-guide/grpc_proxy.md [hunk 3]
```diff
@@ -47,3 +47,31 @@ TODO
...
+|Name|Address|Hostname|
+|------|---------|------------------|
+|infra0|10.0.1.10|infra0.example.com|
+|infra1|10.0.1.11|infra1.example.com|
+|infra2|10.0.1.12|infra2.example.com|
+
+Start the etcd gRPC proxy to use these static endpoints with the command:
+
+```bash
+$ etcd grpc-proxy start --endpoints=infra0.example.com,infra1.example.com,infra2.example.com --listen-addr=127.0.0.1:8080
```
Comment (heyitsanthony) [inline Documentation/op-guide/grpc_proxy.md]: why 8080 instead of 2379?

Comment (xiang90) [inline Documentation/op-guide/grpc_proxy.md]: i should have changed this. i tried on my local machine and use 8080 to avoid port conflict.

**Comment by xiang90** [comment]
@heyitsanthony all fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**